sess/session.go

```diff
@@ -85,7 +85,8 @@ func (s *Session) Abort() error {
 }
 
 // Close must only be invoked if no queries are running
-func (s *Session) Close() error {
+func (s *Session) Close(handler SchemasHandler) error {
```

Comment:
I think it would be better to pass a reference to the metacontroller into the session when creating it here https://github.com/cashapp/pranadb/blob/85447cb420241ccc03f45eda3d1d97b380abf6a1/command/command.go#L187 and then Close() just to use that.

Reply:
Another way would be to provide a session closer

Reply:
If you can get the tests running with how you have it currently, I will merge it and we can make further changes later.

Reply:
done :)